charts/wildfly-common/templates/_buildconfig-s2i-build-artifacts.yaml

```diff
@@ -37,6 +37,14 @@ spec:
       - name: GALLEON_PROVISION_LAYERS
         value: {{ join "," .Values.build.s2i.galleonLayers | quote }}
       {{- end }}
+      {{- if .Values.build.s2i.featurePacks }}
+      - name: GALLEON_PROVISION_FEATUREPACKS
```

Comment:
GALLEON_PROVISION_FEATUREPACKS should be GALLEON_PROVISION_FEATURE_PACKS

Reply:
I can't make a proper copy/paste :( fixed, thanks!